### PR TITLE
Fixes #17941: Create a configure to make the agent package

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/17941

### CODE REVIEW

**Comment by peckpeck** [comment]
Still need som build tests on other systems

**Comment by peckpeck** [comment]
PR updated with a new commit

File: rudder-agent/SOURCES/absolute-aix-path [hunk 1]
```diff
@@ -1,92 +1,23 @@
-#!/bin/bash
```
Comment (amousset) [inline rudder-agent/SOURCES/absolute-aix-path:1]: New content does not match file name.

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**